<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>read_half_data</class>
 <widget class="QDialog" name="read_half_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Критические данные пациента</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Критические данные пациента</string>
   </property>
  </widget>
  <widget class="QLabel" name="AB0_Rh_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Группа крови и резус-фактор:</string>
   </property>
  </widget>
  <widget class="QLabel" name="SD_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сахарный диабет:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AID_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ВИЧ-статус:</string>
   </property>
  </widget>
  <widget class="QLabel" name="H_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Гепатит:</string>
   </property>
  </widget>
  <widget class="QLabel" name="HZ_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Хронические заболевания:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AS_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Непереносимость лекарственных препаратов:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="HZ_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>261</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="AS_list">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>271</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AB0_Rh_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SD_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="AID_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="H_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Read_full">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Получить историю болезни</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
